web/src/app/tokens/tokens-list/tokens-list.component.html

```diff
@@ -1,4 +1,8 @@
 <div class="mat-content">
+  <mat-card class="bread-crumb">
+    <a [routerLink]="['/projects', projectUid]">Projects</a> > <span>Tokens</span>
```

Comment:
I updated the route link to go back to the project the monitor belongs to.

Reply:
okay